exercises/practice/affine-cipher/.docs/instructions.append.md

```diff
@@ -0,0 +1,37 @@
+# Hints
```

Comment:
Going by [python/.../affine-cipher/.../instructions.append.md](https://github.com/exercism/python/blob/59a459abe8ee5e7f2fe69040d4d020284aa6d1a8/exercises/practice/affine-cipher/.docs/instructions.append.md?plain=1) ([rendered](https://exercism.org/tracks/python/exercises/affine-cipher)), if you wish to show a header then you'll need to add an extra level 2 header.